src/perf/bin/drvmain.cpp

```diff
@@ -637,12 +638,11 @@ QuicPerfCtlReadPrints(
             QuicThreadDelete(&Client->Thread);
             Client->Thread = nullptr;
         }
+        WdfRequestUnmarkCancelable(Request);
```

Comment:
Does this return an error that might indicate it's already been canceled, and therefore you shouldn't then complete it again?

Reply:
Also, I see this:
> the driver must not call WdfRequestUnmarkCancelable after EvtRequestCancel calls WdfRequestComplete.
https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/wdfrequest/nf-wdfrequest-wdfrequestunmarkcancelable#remarks